docs/deployment.md

```diff
@@ -4,7 +4,8 @@ title: "☁️ Deployment"
 ---
 
 :::caution Alpha Status
-Chroma Server is currently in Alpha. We are working hard to move Chroma from an in-memory single-process oriented library to a distributed production-grade DB!
+Chroma Server is currently in Alpha. We are working hard to move Chroma from an in-memory single-process oriented
+library to a distributed production-grade DB!
```

Comment:
@tazarov im confused about this indentation changes? was this the linter? (there are a few of them)

Reply:
I think linter, but let me redo it and see what comes out.

Reply:
It's Pycharm cutting down lines to 100ish chars. I think that from an MD rendering perspective, this makes no difference, but it looks weird to look at these odd line breaks.